cql/the-basics-of-cql/cinchy-supported-functions/connections-functions.md

```diff
@@ -4,37 +4,79 @@
 
 The set of functions listed in this page are for use in the [Cinchy Connections Experience](https://cli.docs.cinchy.com/) when configuring data syncs.
 
+* [@COLUMN]
 * [​JSON\_ESCAPE​](connections-functions.md#json\_escape)
 * [​URL\_ESCAPE​](connections-functions.md#url\_escape)
 * [STRING\_ESCAPE()](connections-functions.md#string\_escape)
+* [@ID](connections-functions#id)
+
+## @COLUMN
+
+The `@COLUMN` function is used for real-time syncs where your source is an event (such as Cinchy Event Broker) and your target is REST/SOAP. 
+
+Use `@COLUMN` to build the request body of SOAP/REST API specifications.
+
+### Syntax
+
+```
+{
+    "employeeNane": "@COLUMN('Name')"
```

Comment:
A common gotcha is not providing the "" if you wanted the value to be a string. Not sure if it's worth highlighting or this example is enough